packages/views/organization_detail.jade

```diff
@@ -1,23 +1,48 @@
 template(name="organizationDetail")
   .organization-detail
-    .container
-      .view-header
-        h1= organization.name
-        if currentUser
-          unless userIsMember
-            a.btn.btn-default.header-btn.join-organization
-              span Join
-              i.fa.fa-users
-      +paragraphText text=organization.description
-      .row
-        if memberCount
-          .col-sm-7
-            h2 Members
-              span.member-count (#{memberCount})
-            table.table.member-table
+    section
+      .container
+        if editingName
+          form.edit-name
+            input#organization-name.form-control(type="string" name="name" value=organization.name)
+            .actions.middle
+              button.btn-plain.save.save-name-button(type="submit")
+                i.fa.fa-check-circle
```

Comment:
I was worried about that. I'll change them into buttons.
I suppose, in the future, if there are more fields to edit, we could redirect to an edit page.